config/routes.js

```diff
@@ -1,32 +1,116 @@
-const axios = require('axios');
+const axios = require("axios");
+const bcrypt = require("bcryptjs");
+const dbHelper = require("../database/dbHelper");
+const db = require("../database/dbConfig");
+const jwt = require("jsonwebtoken");
 
-const { authenticate } = require('../auth/authenticate');
+const { authenticate } = require("../auth/authenticate");
 
 module.exports = server => {
-  server.post('/api/register', register);
-  server.post('/api/login', login);
-  server.get('/api/jokes', authenticate, getJokes);
+  server.post("/api/register", register);
+  server.post("/api/login", authenticate, login);
```

Comment:
as creating the token on the login function is the first part of it, i dont think you need to add the authenticate middleware here, its a bit redundant